op-chain-ops/upgrades/l1.go

```diff
@@ -757,9 +757,9 @@ func SystemConfig(batch *safe.Batch, implementations superchain.ImplementationLi
 			L1CrossDomainMessenger:       common.Address(list.L1CrossDomainMessengerProxy),
 			L1ERC721Bridge:               common.Address(list.L1ERC721BridgeProxy),
 			L1StandardBridge:             common.Address(list.L1StandardBridgeProxy),
-			L2OutputOracle:               common.Address(list.L2OutputOracleProxy),
```

Comment:
I think the DGF should be in here: https://github.com/ethereum-optimism/optimism/blob/f2e06c31b8cac95d36f2219a468244163f8bc26f/packages/contracts-bedrock/src/L1/SystemConfig.sol#L38

Reply:
Has the dgf been added to the superchain registry yet? If no, then I would like to save that for the future

Reply:
It has been added for Sepolia:
- https://github.com/ethereum-optimism/superchain-registry/blob/110e744c97b4873384ad2da365c281639fc0668e/superchain/extra/addresses/sepolia/op.json#L12-L14
- https://github.com/ethereum-optimism/superchain-registry/blob/110e744c97b4873384ad2da365c281639fc0668e/superchain/implementations/networks/sepolia.yaml#L15-L16

Reply:
In [09f974b](https://github.com/ethereum-optimism/optimism/pull/10466/commits/09f974b31d3b1b144c3605aa1ff9e0ad0b5c904f) i have added an always error case until the dispute game factory address can be accessed from the `ContractImplementations`, see https://github.com/ethereum-optimism/superchain-registry/blob/110e744c97b4873384ad2da365c281639fc0668e/superchain/superchain.go#L253

Reply:
Opened https://github.com/ethereum-optimism/superchain-registry/pull/227 to address this